fetch2okhttp/src/main/java/com/tonyodev/fetch2okhttp/OkHttpDownloader.kt

```diff
@@ -209,6 +209,10 @@ open class OkHttpDownloader @JvmOverloads constructor(
     }
 
     override fun getRequestSupportedFileDownloaderTypes(request: Downloader.ServerRequest): Set<Downloader.FileDownloaderType> {
+        if(fileDownloaderType == Downloader.FileDownloaderType.SEQUENTIAL) {
```

Comment:
similar to above

Reply:
Hi @AlexStarc  -  thank you for checking out my suggestion there. Yes I think your refactor was more readable (and generally a better style to use in Kotlin). I applied your change to my Pull Request. Let me know if anything else is needed from me.

Reply:
up